distribution.md

```diff
@@ -15,11 +15,12 @@ This document specifies the artifact format, delivery mechanism, and order resol
 
 ## Distribution API Version
 
-This document specifies Distribution API version `0.3`.
+This document specifies Distribution API version `0.4`.
```

Comment:
I removed stacks from the labels below so I think that justifies bumping the spec version?  but I left them in the builder.toml spec as a backwards compat. option for older platform APIs.